README.md

```diff
@@ -47,7 +47,24 @@ described in detail below.
 
 ## 1. Register for a Nimbix Cloud Platform account
 
-Register for Power AI on the [Nimbix Cloud](https://platform.jarvice.com/login-page).
+IBM has partered with Nimbix to provide journey developers a trial
+account that provides 10 hours of free processing time on the Power AI
+platform.
+
+The sign-ip process is as follows:
+
+1) Go to the [IBM Congnitive Journey Demonstration Registration](https://www.nimbix.net/cognitive-journey)
```

Comment:
"Demonstration Registration" sounds funny, can we remove "Demonstration" or shorten it to "Demo"?